extensions/sftp/processors/ListSFTP.cpp

```diff
@@ -870,7 +871,7 @@ void ListSFTP::listByTrackingEntities(
 
   time_t now = time(nullptr);
   uint64_t min_timestamp_to_list = (!initial_listing_complete_ && entity_tracking_initial_listing_target_ == ENTITY_TRACKING_INITIAL_LISTING_TARGET_ALL_AVAILABLE)
```

Comment:
Maybe we can write a separate Jira ticket to replace uint64_t timestamp representations like this to std::chrono in this file.

Reply:
Good idea, I created a Jira so we can collect these leftover todo-s https://issues.apache.org/jira/browse/MINIFICPP-1700